app/controllers/parties_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PartiesController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @movie = MoviesFacade.new.find_movie(params[:movie_id])
```

Comment:
not super important at all, but we could maybe change to `@movie_facade` to match naming patterns we used in the `MoviesController`